src/Extensions/NeoServer.Scripts.LuaJIT/Functions/ActionFunctions.cs

```diff
@@ -165,7 +208,7 @@ public static int LuaActionAllowFarUse(LuaState luaState)
         }
         else
         {
-            ReportError(nameof(LuaActionItemId), GetErrorDesc(ErrorCodeType.LUA_ERROR_ACTION_NOT_FOUND));
+            ReportError(nameof(LuaActionAllowFarUse), GetErrorDesc(ErrorCodeType.LUA_ERROR_ACTION_NOT_FOUND));
```

Comment:
why changing the name to LuaActionAllowFarUse?

Reply:
Because after compared with code base of tfs and identify this methods is equal **AllowFarUse**.